src/news/release-8.3.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Prism Launcher Release 8.3, now available"
+description: Also try Terraria
```

Comment:
Shouldn't we change this?

Reply:
maybe a different splashes.txt entry for this one :D

Reply:
is: `I would wish you good luck, but luck won't carry you through the Nether.` good?